scopes/explorer/ui/base-component-card/details/details.module.scss

```diff
@@ -28,6 +28,15 @@
     margin-right: 5px;
   }
 }
+.versionHolder {
+  display: flex;
+  align-items: center;
+  overflow: hidden;
```

Comment:
why do you need this?

Reply:
![image](https://user-images.githubusercontent.com/35892475/149773152-8bf97de9-2054-4cd7-851a-50e4ece06d9f.png)
Because if I remove the "overflow" the text version will continue.

Reply:
ok